cves/CVE-2009-2693.yml

```diff
@@ -99,9 +99,9 @@ unit_tested:
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
     Must be just "true" or "false".
-  answer:
-  code:
-  fix:
+  answer: No automated tests found in the fix to this vulnerability
```

Comment:
Could you state some reasons why you cannot find any tests? Maybe the commit messages or blog posts not mentioning. Or where you have looked would be fine.